exporter/opentelemetry-exporter-zipkin/setup.cfg

```diff
@@ -39,6 +39,7 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
+    protobuf == 3.12.2
```

Comment:
Must this be an exact version?

Reply:
Doesn't have to be but was mimicking top-level `dev-requirements.txt` to be consistent.

Reply:
dev requirements are quite different. We can enforce exact versions on people contributing to this project but people using this project as a library should have a lot more flexibility.

Reply:
noted - updated